src/lib/Server/Controller/Role.php

```diff
@@ -521,55 +502,16 @@ public function updatePolicy($roleId, $policyId, Request $request)
                     }
                 }
             }
+            throw new BadRequestException($roleDraft->getPolicies()[0]->originalId . '/' . $policyId);
```

Comment:
I'm not convinced about this one. Are you sure `$roleDraft` will always contain at least one policy? Does the error message provide enough verbose information?

Reply:
There is no need for that exception at all, it was some strange leftover ;(